docs/drafts/developer.md

```diff
@@ -1,10 +1,24 @@
+
 ## Getting Started
+
 You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
```

Comment:
The first sentence is confusing as `make run` starts a kind cluster.

Reply:
Lets remove this sentence.

Reply:
I am modifying it to still explain the kind cluster being started and pointing to the manual steps that follow for how to use with a remote cluster.

Reply:
seem ok in 7f54af6c ?